bndtools.api/bnd.bnd

```diff
@@ -10,12 +10,16 @@
 	org.eclipse.core.resources,\
 	org.eclipse.core.runtime,\
 	org.eclipse.equinox.common, \
+	org.eclipse.equinox.registry, \
     org.eclipse.swt,\
 	org.eclipse.swt.cocoa.macosx.x86_64,\
 	org.eclipse.jface
 
 Import-Package: \
  ${eclipse.importpackage},\
+ org.eclipse.debug.ui,\
```

Comment:
This seems wrong. The macro should import the proper packages.

Reply:
Good catch; fixed.